weblate/trans/forms.py

```diff
@@ -1043,8 +1043,7 @@ class UserManageForm(forms.Form):
     user = UserField(
         label=_('User to add'),
         help_text=_(
-            'Please provide username or e-mail. '
-            'User needs to already have an active account in Weblate.'
+            'Please type in a Weblate account name, or any e-mail address. '
```

Comment:
```suggestion
            'Please type in an username or e-mail address of existing Weblate account. '
```

How about this? It should be clear that this is only for existing accounts.